hbase-common/src/main/java/org/apache/hadoop/hbase/trace/TraceUtil.java

```diff
@@ -34,17 +51,24 @@
   private static HTraceConfiguration conf;
   private static Tracer tracer;
 
+  private static io.opentracing.Tracer otTracer;
+
+  private static final Logger LOG = LoggerFactory.getLogger(TraceUtil.class.getName());
+
+
   private TraceUtil() {
   }
 
-  public static void initTracer(Configuration c) {
-    if (c != null) {
-      conf = new HBaseHTraceConfiguration(c);
-    }
+  public static void initTracer(Configuration c, String serviceName) {
+    if (!GlobalTracer.isRegistered()) {
+      io.jaegertracing.Configuration conf = io.jaegertracing.Configuration.fromEnv(serviceName);
```

Comment:
Can we avoid referencing jaeger directly in code? I would expect that this just work like slf4j, when only need to depend on log4j when testing or at runtime, not compile time.